libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -541,6 +541,21 @@ string YulUtilFunctions::roundUpFunction()
 	});
 }
 
+string YulUtilFunctions::divide32CeilFunction()
+{
+	string functionName = "divide_by_32_ceil";
+	return m_functionCollector.createFunction(functionName, [&]() {
+		return
+			Whiskers(R"(
+			function <functionName>(value) -> result {
+				result := div(add(value, 31), 32)
+			}
+			)")
+			("functionName", functionName)
+			.render();
+	});
```

Comment:
Maybe using new interface, but not mandatory
```suggestion
	return m_functionCollector.createFunction(functionName, [&](vector<string>& _args, vector<string>& _returnValues) {
		_args = {"value"};
		_returnValues = {"result"};
		return
			Whiskers(R"(
				result := div(add(value, 31), 32)
			)")
			.render();
	});
```